exercises/concept/high-score/.meta/design.md

```diff
@@ -0,0 +1,14 @@
+# Design
+
+## Considered but dropped
+
+Implement a `high_score` function that returns the leader, e.g.
+
+```julia
+function high_score(scores)
+    best = findmax(scores)
+    best[2] => best[1]
+end
+```
```

Comment:
This doesn't actually work because it doesn't handle the edge cases, I merely left it in as an example for future reference.